src/main/java/com/team01/carsharingapp/controller/PaymentController.java

```diff
@@ -0,0 +1,44 @@
+package com.team01.carsharingapp.controller;
+
+import com.team01.carsharingapp.dto.payment.PaymentDto;
+import com.team01.carsharingapp.dto.payment.PaymentRequestDto;
+import com.team01.carsharingapp.service.PaymentService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/payment")
+public class PaymentController {
+    private final PaymentService paymentService;
+
+    @GetMapping
+    public ResponseEntity<List<PaymentDto>> getPayments(@RequestParam("user_id") Long userId) {
```

Comment:
We don't use ResponseEntity in our project. Please, accord to project rule conventions.